src/main/java/uk/gov/hmcts/darts/dailylist/service/impl/DailyListServiceImpl.java

```diff
@@ -60,20 +58,20 @@ public PostDailyListResponse saveDailyListToDatabase(DailyListPostRequest postRe
 
         DailyListJsonObject dailyList = postRequest.getDailyListJson();
 
-        CourthouseEntity courthouse = retrieveCourtHouse(dailyList);
         String uniqueId = dailyList.getDocumentId().getUniqueId();
         Optional<DailyListEntity> existingRecordOpt = dailyListRepository.findByUniqueId(uniqueId);
         DailyListEntity savedDailyListEntity;
+        CourthouseEntity courthouseEntity = retrieveCourtHouse(dailyList);
```

Comment:
This should just be the `listing_courthouse` string, it's not guaranteed to match a courthouse in our database.